packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/FullscreenButton/FullscreenButton.spec.js

```diff
@@ -0,0 +1,40 @@
+import { shallow, mount } from 'enzyme'
+import React from 'react'
+import sinon from 'sinon'
+
+import FullscreenButton from './FullscreenButton'
+
+describe('Component > FullscreenButton', function () {
+  it('should render without crashing', function () {
+    shallow(<FullscreenButton />)
+  })
+
+  it('should have an ARIA label', function () {
+    const wrapper = mount(<FullscreenButton />)
+    expect(wrapper.find('button').prop('aria-label')).to.equal('View subject in full screen mode')
+  })
+
+  it('should call the enableFullscreen prop function on click if fullscreen is not active', function () {
+    const spy = sinon.spy()
+    const wrapper = mount(
+      <FullscreenButton
+        fullscreen={false}
+        enableFullscreen={spy}
+      />
+    )
+    wrapper.find('button').simulate('click')
+    expect(spy.called).to.equal(true)
```

Comment:
`expect(spy.called).to.be.true` is the preferred BDD syntax with Chai: http://www.chaijs.com/api/bdd/#method_true